.env.local.default

```diff
@@ -1,8 +1,8 @@
-NEXT_PUBLIC_COINBASE_COMMERCE_API_KEY="COMMERCE_API_KEY_1234567890"
+NEXT_PRIVATE_COINBASE_COMMERCE_API_KEY="COMMERCE_API_KEY_1234567890"
 
 NEXT_PUBLIC_GOOGLE_ANALYTICS_ID=GA_TEST_1234567890
 
 # See https://www.coinbase.com/developer-platform/products/base-node
-NEXT_PUBLIC_ONCHAINKIT_API_KEY="CDP_API_KEY_1234567890"
+NEXT_PRIVATE_ONCHAINKIT_API_KEY="CDP_API_KEY_1234567890"
```

Comment:
Is `NEXT_PRIVATE_******` a thing? I think you'd just do `ONCHAIN_API_KEY` and Next envs are private by default

Reply:
They are private by default, in previous projects we've been explicitly defining as private. Fine with either.

Reply:
Feel strongly that we shouldn't do NEXT_PRIVATE, it's not a standard pattern I've seen elsewhere

Reply:
Sounds good, updated